2016-10/october-13.md

```diff
@@ -0,0 +1,43 @@
+## October 13 ([discuss](https://github.com/reactjs/core-notes/pull/33))
+
+### Attendees
+
+* [Dan](https://twitter.com/dan_abramov) (React)
+* [Sebastian](https://twitter.com/sebmarkbage) (React)
+* [Tom](https://twitter.com/tomocchino) (React)
+
+### Team Changes
+
+* [Jim](https://github.com/jimfb) has left Facebook to pursue other projects. Thanks to Jim for all his hard work and contributions!
```

Comment:
Woah no way, how did I miss this! @jimfb I'll miss seeing you in the issue tracker 😢